### PR TITLE
fix check for missing values

### PR DESCRIPTION
**Pull request by kzalite**

### CODE REVIEW

File: frontend-test/unit/workspaceServiceSpec.js [hunk 1]
```diff
@@ -1385,7 +1385,7 @@ define([
     });
 
     describe('checkForMissingValuesInPerformanceTable', function() {
-      it('should return true is there is at least one missing value for both the effect and distribution of a table entry', function() {
+      it('should return true is there both the effect and distribution have missing values', function() {
```
Comment (DanielReid) [inline frontend-test/unit/workspaceServiceSpec.js]: text description is v. weird

File: frontend-test/unit/workspaceServiceSpec.js [hunk 2]
```diff
@@ -1423,6 +1423,30 @@ define([
         var result = workspaceService.checkForMissingValuesInPerformanceTable(performanceTable);
         expect(result).toBeFalsy();
       });
+
+      it('should return true is the effect has a missing value and there is no distribution', function() {
+        var performanceTable = [{
```
Comment (DanielReid) [inline frontend-test/unit/workspaceServiceSpec.js]: is -> if

File: frontend-test/unit/workspaceServiceSpec.js [hunk 3]
```diff
@@ -1423,6 +1423,30 @@ define([
...
+          performance: {
+            distribution: {
+              type: 'empty'
+            }
+          }
+        }];
+        var result = workspaceService.checkForMissingValuesInPerformanceTable(performanceTable);
+        expect(result).toBeTruthy();
+      });
     });
```
Comment (DanielReid) [inline frontend-test/unit/workspaceServiceSpec.js]: missing a test for the false case I think?

Comment (kzalite) [reply frontend-test/unit/workspaceServiceSpec.js]: Second test: 'should return false if each entry has atleast one non-empty effect or distribution'.
Or you think there should be another case?

Comment (DanielReid) [reply frontend-test/unit/workspaceServiceSpec.js]: Apologies, looks like code folding messed me up

**Pull request merged**